device/iot-device-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/device/DeviceClientTest.java

```diff
@@ -147,6 +148,34 @@ public void constructorSuccess() throws URISyntaxException, IOException
         };
     }
 
+    @Test
+    public void constructorWithClientOptionsSuccess() throws URISyntaxException, IOException
+    {
+        // arrange
+        final String connString = "HostName=iothub.device.com;CredentialType=SharedAccessKey;CredentialScope=Device;deviceId=testdevice;SharedAccessKey=adjkl234j52=;";
```

Comment:
is this a real connection string or fake?

Reply:
It is fake may be never changed in our legacy test code. I am removing these tests as they seem unnecessary after rethinking. But this is good to address in follow up PR so I'll keep a note of this.